pkg/finality-grandpa/bridge_state.go

```diff
@@ -13,7 +13,7 @@ type waker struct {
 }
 
 func newWaker() *waker {
-	return &waker{wakeCh: make(chan struct{}, 100_000)}
+	return &waker{wakeCh: make(chan struct{}, 1000)}
```

Comment:
Just making sure the intent was to change this value

Reply:
yea I didn't need to allocate such a large buffered channel.